README.md

```diff
@@ -187,6 +187,37 @@ result = my_zosmf.get_info()
 ```
 The result will be a JSON object containing z/OSMF information
 
+## Local project setup (Windows)
+Enlisted below are the project setup instructions.
+`Note` - Usage of `python3.10` or older is recommended.
+
+
+0. Pre Step - Switch to branch `next`
+```shell
+git checkout next
+```
+
+1. Install a virtual environment package
+
+```shell
+python -m pip install virtualenv
+```
```

Comment:
Is this step required? Python installations after version 3.3 should ship with the `venv` module.